tests/integration/mixer/policy/white_black_listing_test.go

```diff
@@ -71,6 +70,7 @@ func TestWhiteListing(t *testing.T) {
 			t,
 			bookinfoNs,
 			bookinfo.PolicyDenyIPRule.LoadWithNamespaceOrFail(t, bookinfoNs.Name()))
+		util.AllowRuleSync(t)
```

Comment:
aside: it will be nice, in the future, to be able to poll on metrics to see when configuration has been received by Mixer.

Reply:
Agreed.. we can use istio-state-metrics for that purpose I feel..